client/james.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>James Todd</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- I was not sure if removing this portion of the bootstrap and having the stylesheet linked only would cause my bootstrap to not function. I comment it out and it blows the card picture completely out of proportion. -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <link rel="stylesheet" href="../client/assets/css/james.css">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@500&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e45eb01798.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="unify container">
+        <h2>James Todd</h2>
+    </div>
+
+    <div class="card unify container" style="width: 18rem;">
+        <img src="../client/assets/images/Headshot.jpg" class="card-img-top" alt="James Todd">
+        <div class="card-body">
+            <a href="https://jamestodd.netlify.app/">My Portfolio</a>
+        </div>
+    </div>
+    <div class="unify container">
+        <p>Im currently an automotive technician. During my time in the Tech talent south code bootcamp, I was
+            introduced to code and started from the ground up with HTML. My course evolved from html with css into Ruby on rails. The bootcamp left me
+            with hundreds of questions and other then the projects performed in class, no real direction was decided. I have
+            since been able to mentor with Karsten and add React to my arsenal of code. I am currently working on building a PC to handle android app building while teaching myself Java.
+            In my spare time I rock climb, weld, and photography.
+        </p>
+    </div>
+    <div class="unify">
+    <a href="https://github.com/jtodd0515" target="_blank"><i class="fab fa-github-square fa-4x"></i></a>
+    </div>
+    <footer class="unify container">James Todd Copyright 2020</footer>
+    
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
```

Comment:
Can remove the bootstrap Javascript files: 

![James-PR](https://user-images.githubusercontent.com/65138849/92156960-bb1a7600-edf7-11ea-8778-37ad653ee9d6.png)